invenio_app_ils/ui/backoffice/src/pages/UserDetails/state/actions.js

```diff
@@ -0,0 +1,25 @@
+import { IS_LOADING, SUCCESS, HAS_ERROR } from './types';
+import { user } from 'common/api';
+
+export const fetchUserDetails = userPid => {
```

Comment:
`userPid` -> `userId`? Sorry if I am picky, but I struggled a lot when learning Invenio to distinguish what is a PID and what is an ID, and I would personally prefer we call them with the right name to avoid confusion.

Reply:
👍 I will rename it